lib/supply/client.rb

```diff
@@ -231,6 +274,28 @@ def upload_apk_to_track_with_rollout(path_to_apk, track, rollout)
       raise result_update.error_message.red if result_update.error?
     end
 
+    def update_apk_listing_for_language(apk_listing)
+      ensure_active_edit!
+
+      body_object = {
+        'language' => apk_listing.language,
+        'recentChanges' => apk_listing.recent_changes
+      }
+
+      result = api_client.execute(
+        api_method: android_publisher.edits.apklistings.update,
+        parameters: {
+          'apkVersionCode' => apk_listing.apk_version_code,
+          'editId' => current_edit.data.id,
+          'packageName' => current_package_name,
+          'language' => apk_listing.language
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.